README.md

```diff
@@ -68,13 +68,13 @@ Example   | Phase | Description |
 
 **Mondoo Scripts**
 
-- [install.sh](./install.sh) - Mondoo Agent Bash Installer for Servers
-- [download.sh](./download.sh) - Mondoo Agent Bash Downloader for Workstation
+- [install.sh](./install.sh) - Mondoo Client Bash Installer for Servers
+- [download.sh](./download.sh) - Mondoo Client Bash Downloader for Workstation
 - [Dockerfile](./Dockerfile) - Build script for official Mondoo container
 
 **Docker Containers**
 
-- https://hub.docker.com/r/mondoolabs/mondoo
+- https://hub.docker.com/r/mondoo/client
```

Comment:
can we add the default cnquery and cnspec imaged and make the mondoo image last?

Reply:
There's a much larger amount of work to do in order to get the whole repo realigned for cnspec/cnquery. Just changing this URL would leave it in a 1/2 done state. It should get done in its own set of work, which can happen when we flip the console/docs over to cnspec/cnquery (soon)